tests/ctst/steps/replication.ts

```diff
@@ -112,12 +115,22 @@ Then('the object should eventually {string} replicated', { timeout: 360_000 },
             assert(parsed.ok);
             const replicationStatus = parsed.result?.ReplicationStatus;
             
-            if (replicate === 'be') {
+            if ((expectedOutcome === 'succeed' || expectedOutcome === 'fail') && 
+                (replicationStatus === 'PENDING' || 
+                    replicationStatus === 'PROCESSING' ||
```

Comment:
this looks kind of weird next the switch on status: for better consistency & readability, either
* we should move the condition in pending in the `expectedOutcome` switch;
* or we should do a switch on the status, which should have less cases I think

```suggestion
switch(replicationStatus) {
case 'PENDING', 'PROCESSING':
   assert(expectedOutcome != 'never happen');
   break;

case 'COMPLETED':
   assert(expectedOutcome === 'succeed');
   return

case 'FAILED':
   assert(expectedOutcome === 'fail');
   break;

case undefined:
   assert(expectedOutcome == 'never happen');
   break; // or return if we don't need to wait the timeout
}
```

Reply:
Yes, I changed the switch to something similar to this